Onboarding/onboarding-part-1-profile-classfier.md

```diff
@@ -0,0 +1,574 @@
+end
+
+```
+
+## Domains 5
+
+```stack
+card Domains5, then: Domains5Branch do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_domains_05"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = page.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
+end
+
+# Text only
+card Domains5Branch when @contact.data_preference == "text only" do
+  buttons(AddDomain5: "@button_labels[0]", Domains6: "@button_labels[1]") do
+    text("@message.message")
+  end
+end
+
+# Show image
+card Domains5Branch do
+  image_id = page.body.body.text.value.image
+
+  image_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/images/@image_id/",
+      headers: [
+        ["Authorization", "Token @config.items.contentrepo_token"]
+      ]
+    )
+
+  buttons(AddDomain5: "@button_labels[0]", Domains6: "@button_labels[1]") do
+    image("@image_data.body.meta.download_url")
+    text("@message.message")
+  end
+end
+
+card AddDomain5, then: Domains6 do
+  update_contact(well_being: "true")
+end
+
+```
+
+## Domains 6
+
+```stack
+card Domains6, then: Domains6Branch do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_domains_06"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = page.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
+end
+
+# Text only
+card Domains6Branch when @contact.data_preference == "text only" do
+  buttons(AddDomain6: "@button_labels[0]", Domains7: "@button_labels[1]") do
+    text("@message.message")
+  end
+end
+
+# Show image
+card Domains6Branch do
+  image_id = page.body.body.text.value.image
+
+  image_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/images/@image_id/",
+      headers: [
+        ["Authorization", "Token @config.items.contentrepo_token"]
+      ]
+    )
+
+  buttons(AddDomain6: "@button_labels[0]", Domains7: "@button_labels[1]") do
+    image("@image_data.body.meta.download_url")
+    text("@message.message")
+  end
+end
+
+card AddDomain6, then: Domains7 do
+  update_contact(family_planning: "true")
+end
+
+```
+
+## Domains 7
```

Comment:
It's odd that the slugs for these were chosen as `domains_07`, instead of `domains_info_for_health_professionals`, I would've preferred the naming of the stacks to refer to what they are instead, but I'm happy keeping it this way for consistency across the content and figma flows.

Reply:
Same. I'll add some additional context to the docs sections in my next PR for this